RecoLocalCalo/HGCalRecProducers/plugins/HGCalLayerClusterProducer.cc

```diff
@@ -101,6 +110,7 @@ void HGCalLayerClusterProducer::fillDescriptions(edm::ConfigurationDescriptions&
   desc.add<edm::ParameterSetDescription>("plugin", pluginDesc);
   desc.add<std::string>("detector", "all");
```

Comment:
please add a comment here to clarify that all does not include HFNose and possibly list all other variants
in ```->setComment("all (does not include HFNose); other options: EE, FH, HFNose; other value defaults to BH");```